src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,47 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+  }
+
+  static adder(instance) {
+    this.alive.push(instance);
+  }
```

Comment:
You can add here name, and delete all Animal.adder(this);

```suggestion
  constructor(name) {
    this.name = name;
    this.health = 100;
    
    this.alive.push(name);
  }
```